deployment/volumes/volume-snapshots/neo4j-data.yaml

```diff
@@ -2,7 +2,7 @@
   kind: PersistentVolumeClaim
   apiVersion: v1
   metadata:
-    name: neo4j-data-claim
+    name: YYYY-MM-DD-neo4j-data-claim
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-08T13:18:56Z" title="Wednesday, April 8th 2020, 3:18:56 pm +02:00">Apr 8, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -2,7 +2,7 @@
   kind: PersistentVolumeClaim
   apiVersion: v1
   metadata:
-    name: neo4j-data-claim
+    name: YYYY-MM-DD-neo4j-data-claim
```


see :point_up_2: 
this is to set up a new persistent volume claim and persistent volume, which can be seen by running 
```
kubectl get pvc -n human-connection
```
and 
```
kubectl get pv -n human-connection
```
completely different from `volumesnapshots`

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-04-15T09:16:54Z" title="Wednesday, April 15th 2020, 11:16:54 am +02:00">Apr 15, 2020</time>_

---


Same same ☝🏼

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-16T06:27:18Z" title="Thursday, April 16th 2020, 8:27:18 am +02:00">Apr 16, 2020</time>_

---


the claim name is not unique and therefore does not need to have the date in front of it. The persistent volume claims in development, staging, and production all have the same names `neo4j-data-claim`. We do not want to edit this file.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-04-16T07:22:12Z" title="Thursday, April 16th 2020, 9:22:12 am +02:00">Apr 16, 2020</time>_

---


Okay. Now I got the point!

And `kubectl get pv -n human-connection` don’t need the name space! @mattwr18 😂